tests/docker/Dockerfile

```diff
@@ -150,4 +150,15 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
     cp target/release/client-swarm /usr/local/bin && \
     cd .. && rm -rf client-swarm && rm -rf /root/.cargo
 
+# Seastar addrress to line utility
+RUN apt update && \
+    apt install -y \
+      file && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /opt/scripts && \
+    curl https://raw.githubusercontent.com/redpanda-data/seastar/master/scripts/addr2line.py -o /opt/scripts/addr2line.py && \
```

Comment:
Let's use static sha1s in these URLs to avoid breaking in future

Reply:
good point, i was going to change that and then forgot, will do